docker-compose.yml

```diff
@@ -5,11 +5,11 @@ server:
   volumes:
    - .:/src
    - /var/run/docker.sock:/tmp/docker.sock
-   - ./config.yml:/config.yml
+   - ~/volumes/ci/roger:/roger/
```

Comment:
you cant force people to create directories in their home folder when they're just experimenting with some new tool -- I dont think we can merge this change as it should be either documented in the README or, better, simply use a directory inside the repo that acts as a sample